.ci/env/openblas.sh

```diff
@@ -46,23 +76,49 @@ done
 target=${target:-ARMV8}
 host_compiler=${host_compiler:-gcc}
 compiler=${compiler:-aarch64-linux-gnu-gcc}
-cflags=${cflags:--march=armv8-a+sve}
+
+target_arch=${target_arch:-$(uname -m)}
+blas_prefix="${ONEDAL_DIR}/__deps/openblas_${target_arch}"
 
 sudo apt-get update
 sudo apt-get -y install build-essential gcc gfortran
-git clone https://github.com/xianyi/OpenBLAS.git
-CoreCount=$(lscpu -p | grep -Ev '^#' | wc -l)
-pushd OpenBLAS
-  make clean
+blas_src_dir=${blas_src_dir:-$OPENBLAS_DEFAULT_SOURCE_DIR}
+if [[ ! -d "${blas_src_dir}" ]] ; then
+  git clone https://github.com/xianyi/OpenBLAS.git "${blas_src_dir}"
```

Comment:
Can we change this to the latest updated link? `https://github.com/OpenMathLib/OpenBLAS`

Reply:
Yeah, absolutely